packages/api-generator/src/maps/v-data-table.js

```diff
@@ -43,6 +43,46 @@ const DataTableEvents = [
     source: 'v-data-table',
     value: `MouseEvent, ${dataString}`,
   },
+  {
+    name: 'mousedown:row',
+    source: 'v-data-table',
+    value: `${dataString}, MouseEvent`,
+  },
+  {
+    name: 'mousemove:row',
+    source: 'v-data-table',
+    value: `${dataString}, MouseEvent`,
+  },
+  {
+    name: 'mouseup:row',
+    source: 'v-data-table',
+    value: `${dataString}, MouseEvent`,
+  },
+  {
+    name: 'mouseenter:row',
+    source: 'v-data-table',
+    value: `${dataString}, MouseEvent`,
+  },
+  {
+    name: 'mouseleave:row',
+    source: 'v-data-table',
+    value: `${dataString}, MouseEvent`,
+  },
+  {
+    name: 'touchstart:row',
+    source: 'v-data-table',
+    value: `${dataString}, TouchEvent`,
+  },
+  {
+    name: 'touchmove:row',
+    source: 'v-data-table',
+    value: `${dataString}, TouchEvent`,
+  },
+  {
+    name: 'touchend:row',
+    source: 'v-data-table',
+    value: `${dataString}, TouchEvent`,
+  },
```

Comment:
I don't know if this is really necessary considering it takes any dom event now.